src/components/mainLayout/mainHeader/MainHeader.module.css

```diff
@@ -0,0 +1,254 @@
+.headerWrap {
+    background-color: var(--color-pure-light);
+}
+
+.headerContainer {
+    width: 93%;
+    display: flex;
+    justify-content: space-between;
+    padding: 46px 0;
+    margin: 0 auto;
+    align-items: center;
+}
```

Comment:
width 100%
max-width 1760px...328px in media queries
this option is more universal and creates less errors with making layout